README.md

```diff
@@ -41,61 +41,62 @@ With this FAQ, we want to cover the most important questions related to the usag
 ## [ProtonDB] The "Run" and "Install" buttons doesn't seem to work on Firefox.
 If Firefox doesn't ask you how it should proceed with `steam://` URLS, you need to force Firefox to do so. For that open `about:config` in your Firefox, right click anywhere in the list and select `New` » `Boolean`. Enter `network.protocol-handler.expose.steam` as the name for the new entry, and `false` as the value. Now if you click on a `steam://` URL, Firefox should ask you how to proceed.
 
-## [ProtonDB] At the site my game is displayed as a windows game but it have a native port or the other way round.
+## [ProtonDB] On Steam my game is displayed as a windows game, but it has had a native port before!
 
-ProtonDB obtains the data from SteamDB and these in turn from the developers of the respective game. So if the developers had a native port like for example at Rust and it was closed then the game will be shown as a Windows game but buyers at the time of the native port will keep the native port of course so there will be some users who write: "it have a native port!".
+ProtonDB obtains the data from SteamDB, and these in turn are set by the developers of the respective game you want to play. So if the developers had a native port like, for example; Rust, and it was closed, then the game will be shown as a Windows game, but buyers who purchased the native port will keep said port as an option in their Steam library, so there will be some users who write: "it has a native port!", when that sometimes is no longer the case.
 
-## Why do my games crash on start, run very slow or have rendering issues?
+## Why do my games crash on startup, run very slowly or have rendering issues?
 
-#### Be sure that your system is up-to-date and that you are using the latest drivers available your graphics card. You will find detailed informations about how to update your graphic drivers in our [Wiki](https://github.com/NoXPhasma/protondb_faq/wiki/Graphics-driver-installation)
+#### Be sure that your system is up-to-date and that you are using the latest drivers available for your graphics card. Generally, for AMD and Intel, you should have be using a recent version of Mesa (19+) if you're on AMD or Intel, and Nvidia users should insteall the proprietary drivers from Nvidia's site:  You will find more detailed information about how to update your graphic drivers on our [Wiki](https://github.com/NoXPhasma/protondb_faq/wiki/Graphics-driver-installation)
```

Comment:
Its not recommended to install nvidia from nvidia's site.

Reply:
Done.